lib/membrane_rpicam_plugin/source.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.Rpicam.Source do
+  @moduledoc """
+  Membrane Source Element for capturing live feed from a RasperryPi camera using rpicam-apps based on libcamera
+  """
+
+  use Membrane.Source
+  alias Membrane.{Buffer, H264, RemoteStream}
+
+  @app_name "libcamera-vid"
+
+  def_output_pad :output,
+    accepted_format: %RemoteStream{type: :bytestream, content_format: H264},
+    availability: :always,
+    flow_control: :push
+
+  def_options timeout: [
+                spec: non_neg_integer() | :infinity,
+                default: :infinity,
+                description: """
+                Time for which program runs in milliseconds.
+                """
+              ],
+              framerate: [
+                spec: {pos_integer(), pos_integer()},
+                default: {-1, 1},
+                description: """
+                Fixed framerate.
+                """
+              ],
+              width: [
+                spec: non_neg_integer(),
+                default: 0,
+                description: """
+                Output image width.
+                """
+              ],
+              height: [
+                spec: non_neg_integer(),
+                default: 0,
+                description: """
+                Output image height.
+                """
+              ]
+
+  @impl true
+  def handle_playing(_ctx, state) do
+    stream_format = %RemoteStream{type: :bytestream, content_format: H264}
+
+    port = Port.open({:spawn, create_command(state)}, [:binary, :exit_status])
```

Comment:
Consider using https://github.com/fhunleth/muontrap for the reasons described [in the readme](https://github.com/fhunleth/muontrap#the-problem)

Reply:
I wasn't able to locate functionality of capturing stdout, since i believe muontrap only implements System.cmd behavior

Reply:
Right, let's leave it then, at least for now